development/kops/run_all.sh

```diff
@@ -13,26 +13,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+set -exo pipefail
+BASEDIR=$(dirname "$0")
+
 function cleanup()
 {
   echo 'Deleting...'
-  ./delete_cluster.sh || true
-  ./delete_store.sh
+  ./${BASEDIR}/delete_cluster.sh || true
+  ./${BASEDIR}/delete_store.sh
   exit 255;
 }
 
-set -exo pipefail
-BASEDIR=$(dirname "$0")
 echo "This script will create a cluster, run tests and tear it down"
 cd "$BASEDIR"
-source ./set_environment.sh
+source ./${BASEDIR}/set_environment.sh
 $PREFLIGHT_CHECK_PASSED || exit 1
-./install_requirements.sh
-trap cleanup SIGINT SIGTERM ERR EXIT
-./create_values_yaml.sh
-./create_configuration.sh
-./create_cluster.sh
-./set_nodeport_access.sh
-./cluster_wait.sh
-./validate_eks.sh
-./run_sonobuoy.sh
+./${BASEDIR}/install_requirements.sh
+trap cleanup SIGINT SIGTERM ERR
+./${BASEDIR}/create_values_yaml.sh
```

Comment:
Is BASEDIR an absolute path? Is the `./` prefix neccessary?

Reply:
BASEDIR is the relative path to the file directory.